### DISCUSSION TITLE
Published internal package to npm in a private org, now cannot install internal version

### DISCUSSION THREAD

**Discussion by mrwillis** [vercel/turborepo#3679] [Help] [answered]
I accidentally published a scoped internal package to npm. My apps have a dependency like

```json
  "dependencies": {
    "@org/repo": "*",
  },
  ```
  
  following this 
  
  https://github.com/vercel/turbo/discussions/315
  
I've unpublished the package from npm, but everytime I try to do `yarn install` in the root now it tries to fetch the package from npm instead of building locally, giving the error

"Couldn't find package on npm registry" 

How do I tell turborepo to look for the package locally again :S

Thanks

Error:

```
Error: Couldn't find package "@org/BROKEN_PACKAGE*" required by "@org/utils@0.0.0" on the "npm" registry.
    at MessageError.ExtendableBuiltin (/Users/julian/.volta/tools/image/yarn/1.22.19/lib/cli.js:721:66)
    at new MessageError (/Users/julian/.volta/tools/image/yarn/1.22.19/lib/cli.js:750:123)
    at PackageRequest.<anonymous> (/Users/julian/.volta/tools/image/yarn/1.22.19/lib/cli.js:36590:17)
    at Generator.throw (<anonymous>)
    at step (/Users/julian/.volta/tools/image/yarn/1.22.19/lib/cli.js:310:30)
    at /Users/julian/.volta/tools/image/yarn/1.22.19/lib/cli.js:323:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    ```

**Comment by frencojobs** [comment, marked as answer]
PNPM has `workspace:*` to specify that a dependency should be resolved by the workspace. Yarn v1 only has `*` but it should look for the local one first before jumping to the registry. Try again after clearing the cache (remove node_modules, and yarn.lock etc). If it's still not working even after that, you might be able to specify a different version (than the one you've published) and get it working. See https://github.com/yarnpkg/yarn/issues/4878#issuecomment-386607832. If all else failed, you can link the dependency like this and that shouldn't fail.

```json
"@org/package": "link:../../package"
```
